apps/connection/src/connection.repository.ts

```diff
@@ -149,7 +149,7 @@ export class ConnectionRepository {
           break;
       }
 
-      const agentDetails = await this.prisma.connections.upsert({
+      const connectionDetails = await this.prisma.connections.upsert({
```

Comment:
You can directly return the result without assigning it to a variable:

```suggestion
      return this.prisma.connections.upsert({
```

Reply:
Resolved